setup.py

```diff
@@ -27,7 +27,8 @@
         'funcsigs',
         'mypy_extensions',
         'typed_ast',
-        'jsonschema'
+        'jsonschema',
+        'spacy'
```

Comment:
Add a keyword in extra requires and place spacy there.

Reply:
👍